streamparse/cli/tail.py

```diff
@@ -5,32 +5,44 @@
 from __future__ import absolute_import, print_function
 
 from fabric.api import env, execute, parallel, run
+from pkg_resources import parse_version
 
-from .common import add_environment, add_name, add_pattern
-from ..util import activate_env, get_logfiles_cmd, get_topology_definition
+from .common import (add_environment, add_name, add_override_name, add_pattern,
+                     add_pool_size)
+from ..util import (activate_env, get_env_config, get_logfiles_cmd,
+                    get_topology_definition, get_nimbus_client,
+                    nimbus_storm_version, ssh_tunnel)
 
 
 @parallel
-def _tail_logs(topology_name, pattern, follow, num_lines):
+def _tail_logs(topology_name, pattern, follow, num_lines, is_old_storm):
```

Comment:
This is scope creep, but by modifying `_remove_logs` to take a user, it seems like this function should also take a user. Adding a `user` parameter to `_remove_logs` acknowledges that the user account SSHing to the machine may not have privileges to delete the file, but it's also possible in that environment the SSH user may not be able to read the files either. 

For consistency this should eventually get a `user` option, but it's not required since it's far more likely the SSH user can at least read the logs.

Reply:
Yeah, adding that in the future makes sense.